ml_tools/eolearn/tests/test_train_split.py

```diff
@@ -63,13 +65,17 @@ def test_train_split():
         assert unique_counts[0] == expected_count
 
     # seed=2 should produce different result than seed=1
-    patch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type="per_class")(patch, seed=2)
+    patch = TrainTestSplitTask(INPUT_FEATURE, OUTPUT_FEATURE, bins, split_type=TrainTestSplitType("per_class"))(
```

Comment:
since it's an Enum, you should use `TrainTestSplitType.PER_CLASS`

Reply:
done